soc/arm/infineon_xmc/4xxx/Kconfig.series

```diff
@@ -0,0 +1,16 @@
+# XMC4xxx MCU series
+#
+# Copyright (c) 2020 Linumiz
+# Author: Parthiban Nallathambi <parthiban@linumiz.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_XMC_4XXX
+	bool "XMC 4xxx Series MCU"
+	select ARM
+	select CPU_CORTEX_M4
+	select CPU_CORTEX_M_HAS_DWT
+	select SOC_FAMILY_XMC
+	select CPU_HAS_ARM_MPU
```

Comment:
Question @parthitce does this CPU have an FPU (floating point unit)? If so , please add this as well.

Reply:
CPU_HAS_FPU was defined at Kconfig.soc and I just realized that all the XMC4xxx series is based on M4F. Moved to Kconfig.series. Thanks.